src/mono/sample/iOS/Makefile

```diff
@@ -82,14 +89,14 @@ run-catalyst: clean appbuilder
 
 run-catalyst-interp: clean appbuilder
 	$(DOTNET) publish \
-	-c $(MONO_CONFIG) \
+	-c $(BUILD_CONFIG) \
 	/p:TargetOS=maccatalyst \
-	/p:TargetArchitecture=$(MONO_ARCH) \
+	/p:TargetArchitecture=$(TARGET_ARCH) \
 	/p:MonoEnableLLVM=false \
 	/p:MonoForceInterpreter=true \
 	/p:DeployAndRun=$(DEPLOY_AND_RUN) \
 	/p:EnableAppSandbox=$(APP_SANDBOX) \
 	/bl
 
 clean:
-	rm -rf bin
+	rm -rf obj bin *.binlog
```

Comment:
nit: I might be wrong, but if I remember correctly the intermediate dir for the Mono sample will be in artifacts folder. 
For NativeAOT iOS sample we did not set it up correctly so we are creating both bin and obj next to the project file.

Reply:
I checked and we have the folders in the current directory too.